From 7b45766152d2600279129722331df380f70a57ee Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kristian=20H=C3=B8gsberg?= <krh@bitplanet.net>
Date: Fri, 13 Sep 2013 15:16:31 -0700
Subject: [PATCH 27/38] os: Also define ListenOnOpenFD and AddClientOnOpenFD
 unconditionally

Remove the #ifdef from the header file as well.
---
 include/os.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/include/os.h b/include/os.h
index 2a24f4b..542ca64 100644
--- a/include/os.h
+++ b/include/os.h
@@ -166,10 +166,9 @@ extern _X_EXPORT void MakeClientGrabImpervious(ClientPtr /*client */ );
 
 extern _X_EXPORT void MakeClientGrabPervious(ClientPtr /*client */ );
 
-#if defined(XQUARTZ) || defined(XORG_WAYLAND)
 extern _X_EXPORT void ListenOnOpenFD(int /* fd */ , int /* noxauth */ );
+
 extern _X_EXPORT void AddClientOnOpenFD(int /* fd */ );
-#endif
 
 extern _X_EXPORT CARD32 GetTimeInMillis(void);
 extern _X_EXPORT CARD64 GetTimeInMicros(void);
-- 
1.8.3.1

